src/test/java/com/google/api/codegen/gapic/testdata/csharp/csharp_library.baseline

```diff
@@ -2152,24 +3305,314 @@ namespace Google.Example.Library.V1.Samples
             LibraryServiceClient libraryServiceClient = LibraryServiceClient.Create();
             PublishSeriesRequest request = new PublishSeriesRequest
             {
-                Shelf = new Shelf(),
-                Books = { },
-                SeriesUuid = new SeriesUuid(),
-            };
-            PublishSeriesResponse response = libraryServiceClient.PublishSeries(request);
-            // testing writing bytes field.
-            File.WriteAllBytes($"uuid_of_series_with_book_{response.BookNames[0]}.raw", response.SeriesUuid.SeriesBytes.ToArray());
-            // testing writing string field.
-            File.WriteAllText("series_uuid_in_plain_text.txt", response.SeriesUuid.SeriesString);
+                Shelf = new Shelf(),
```

Comment:
I'm unclear on what's changed in these first 9 lines of this change?

Reply:
It seems these are just github getting confused ;-|